cli/src/pcluster/models/cluster.py

```diff
@@ -511,6 +535,14 @@ def _upload_artifacts(self):
             LOGGER.error(message)
             raise _cluster_error_mapper(e, message)
 
+    def _get_yaml_path(self, path, parameter):
```

Comment:
I'd move this method into the `config_patch` module, it's more related to the changes structure rather than the `Cluster` class.

Reply:
done